agent/bench-scripts/unittests

```diff
@@ -72,7 +72,7 @@ function remove_path {
 PATH=$(remove_path /opt/pbench-agent/bench-scripts ${PATH})
 PATH=$(remove_path /opt/pbench-agent/util-scripts ${PATH})
 # Allows us to intercept scp, ssh, rsync, etc.
-export PATH=${_testopt}/unittest-scripts:${_testopt}/util-scripts:${_testopt}/bench-scripts:${PATH}
+export PATH=${_testopt}/unittest-scripts:${_testopt}/util-scripts:${_testopt}/bench-scripts:${PWD}/../../bin:${PATH}
```

Comment:
Using $PWD is going to be unreliable I think: for the unittests, we'll have to copy the new hierarchy (@portante suggested `common/bin` and `common/lib`, rather than the bare `bin` and `lib` I had suggested in a previous comment) into the test directory. PATH should be set from there, not from $PWD.

Reply:
ok